rules/integrations/cloud_defend/lateral_movement_ssh_process_launched_inside_a_container.toml

```diff
@@ -45,7 +45,7 @@ type = "eql"
 
 query = '''
 process where container.id: "*" and event.type== "start" and
-event.action in ("fork", "exec") and event.action != "end" and
```

Comment:
Not sure if the Event aggregation affects Defend for Cloud, but I'm removing `event.action != "end"` as it would never match anyway as `event.action in ("fork", "exec")` is specified